### PR TITLE
Configuration Complete

### PR DESCRIPTION
**Pull request by M-Shehu**
Configured ESLint, Circle CI, Jest, Enzyme and passed my Circle CI test.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ jobs:
   build:
     working_directory: ~/hrsf114-front-end-capstone
     docker:
-      - image: circleci/node:4.8.2
+      - image: circleci/node:10.0.0
```
Comment (sebastianmward419) [inline .circleci/config.yml]: Good job for changing the node version

File: .circleci/config.yml [hunk 2]
```diff
@@ -12,7 +12,7 @@ jobs:
           command: 'sudo npm install -g npm'
       - run:
           name: install-dependencies
-          command: 'npm run build'
+          command: 'npm install'
```
Comment (sebastianmward419) [inline .circleci/config.yml]: Why did you change from npm run build to npm install

File: .eslintrc.js [hunk 3]
```diff
@@ -1,10 +1,6 @@
-/**
- * These rules enforce the Hack Reactor Style Guide
- *
- * Visit this repo for more information:
- *   https://github.com/reactorcore/eslint-config-hackreactor
- */
-
 module.exports = {
-  extends: './node_modules/eslint-config-hackreactor/index.js'
+  extends: 'airbnb-base',
```
Comment (sebastianmward419) [inline .eslintrc.js]: Good job for changing to the airbnb style guide

File: .eslintrc.js [hunk 4]
```diff
@@ -1,10 +1,6 @@
...
+  "rules": {
+    "max-len": ["error", { "ignoreComments": true }]
+  }
```
Comment (sebastianmward419) [inline .eslintrc.js]: Why did you add this code to your eslint config file?

File: .circleci/config.yml (hunk 1 above)
Comment (M-Shehu) [reply .circleci/config.yml]: Thank you

File: .circleci/config.yml (hunk 2 above)
Comment (M-Shehu) [reply .circleci/config.yml]: The correct way to install dependencies is npm install

File: .eslintrc.js (hunk 3 above)
Comment (M-Shehu) [reply .eslintrc.js]: Thank you

File: .eslintrc.js (hunk 4 above)
Comment (M-Shehu) [reply .eslintrc.js]: Because ESLint picks up comments longer than 65. For descriptive comments, the Linter might pick it up as an error
